cmake/modules/RootMacros.cmake

```diff
@@ -370,29 +470,38 @@ function(ROOT_GENERATE_DICTIONARY dictionary)
   endif()
 
   if(CMAKE_PROJECT_NAME STREQUAL ROOT)
-    set(includedirs -I${CMAKE_SOURCE_DIR}
-                    -I${CMAKE_BINARY_DIR}/etc/cling/ # This is for the RuntimeUniverse
-                    -I${CMAKE_BINARY_DIR}/include)
+    list(APPEND includedirs -I${CMAKE_BINARY_DIR}/include)
```

Comment:
I thought the intent was to get rid of this?

Reply:
Old code